pkg/apis/build/v1alpha1/build_types.go

```diff
@@ -41,14 +41,14 @@ var (
 	_ kmeta.OwnerRefable = (*Build)(nil)
 )
 
-type BuilderImage struct {
+type BuildBuilderSpec struct {
```

Comment:
Not a fan of the name due to the repetition. Unfortunately not sure what to suggest there